2024/wg-fedid.html

```diff
@@ -270,12 +257,12 @@ <h3>Tentative Deliverables</h3>
         <h3>
           Other Deliverables
         </h3>
-        <p>Other non-normative documents will be created, such as:</p>
+        <p>Other non-normative documents will be created, including:</p>
         <ul>
           <li>A test suite, available from <a
-          href="https://github.com/web-platform-tests/wpt">web-platform-tests</a> as possible, must
-          be created.</li>
-        <li>A deliverable (or set of deliverables) considering the threats and mitigations of Digital Credentials-related technologies concerning human rights, harm, security, and privacy. These findings will be used as input for our deliverables. This will be developed in collaboration with W3C's Technical Architecture Group (TAG), Privacy Interest Group (PING), and other relevant groups.</li>
+          href="https://github.com/web-platform-tests/wpt">web-platform-tests</a>, will
+          be created for each normative specification.</li>
+          <li>A deliverable considering the threats and mitigations of Digital Credentials-related technologies concerning security, privacy, and human rights. These findings will be used as input for any of the group's Digital Credentials deliverables. This will be developed in collaboration with W3C's Technical Architecture Group (TAG), Privacy Interest Group (PING), Credentials Community Group and other relevant groups.</li>
```

Comment:
```suggestion
          <li>A deliverable considering the threats and mitigations of Digital Credentials-related technologies concerning security, privacy, and human rights. These findings will be used as input for any of the group's Digital Credentials deliverables. This will be developed in collaboration with W3C's Technical Architecture Group (TAG), Privacy Interest Group (PING), Verifiable Credentials Working Group (VCWG) and other relevant groups.</li>
```